### DISCUSSION TITLE
Etesync Dav Setup

### DISCUSSION THREAD

**Discussion by 2br-2b** [mikeberger/borg_calendar#100] [Q&A]
Hello!

I am trying to set up my Etesync calendar to work with Borg calendar via their [Etesync Dav Bridge](https://github.com/etesync/etesync-dav/). I keep getting this error:
```
Syncing....Please wait
SYNC: connect to http://localhost:37358/
net.fortuna.ical4j.connector.ObjectStoreException: net.fortuna.ical4j.connector.FailedOperationException: Principals not found at [//cal.php/calendars//BroadsideDeductGettingExtinct]
done
```

My configuration is shown below:

![screenshot](https://user-images.githubusercontent.com/43968991/112324682-6c480700-8c89-11eb-9148-dbf4be987924.png)

I'm not really sure what I'm doing, but I want to try using Borg calendar! Thank you!

**Comment by mikeberger** [comment]
I can't run etesync, but my best guess would be the following:

Principal Path - leave empty
User Path - leave empty
CALDAV User - your username on etesync
CALDAV Calendar - the name of your calendar (if you never named a calendar, it could be default, but not all servers use "default" as the default name)

You'll likely just get a similar error. The third-party library that I use to connect to caldav servers is quite dated does not have the capability to figure out the proper way to connect given just the base server name.
